README.md

```diff
@@ -115,6 +117,41 @@ be available on their page, or you can visit http://localhost:6543/?embed=1&cont
 | React development server   | [`6543`](http://localhost:6543) |
 | Gunicorn for Django app    | [`8081`](http://localhost:8081) |
 
+
+### Parallel Development of OGR
+
+The Open Goods Registry is the successor to OAR and OGR will replace OAR when it
+is launched. Durning the beda period both applications will be deployed and
+development in parallel. Fetures development for OAR also need to be included in
+OGR, bit there are OGR specific features and releated architectural changes that
+are not appropriate to be included in OAR. Too support this parallel development
+process there are 4 main branches in the repository:
+
+- develop
+- master
+- ogr/develop
+- ogr/master
+
+#### Parallel PR Workflow
```

Comment:
It would be nice to add a version of this to the `PULL_REQUEST_TEMPLATE.md` - maybe specifically some new line-items in the checklist.

Reply:
Good idea. I added the following two PR checklist items in fixup 40f1e05a

```
- [ ] This PR is targeted at the correct branch (`develop` vs. `ogr/develop`)
- [ ] If this PR applies to both OAR and OGR a companion PR has been created
```